packages/ariakit/src/focus-trap/__examples__/focus-trap-region/index.tsx

```diff
@@ -0,0 +1,31 @@
+import "./style.css";
+import { Button } from "ariakit/button";
+import { Checkbox, useCheckboxState } from "ariakit/checkbox";
+import { FocusTrapRegion } from "ariakit/focus-trap/focus-trap-region";
+
+export default function Example() {
+  const focusTrapped = useCheckboxState({ defaultValue: true });
+
+  return (
+    <>
+      <FocusTrapRegion className="wrapper" enabled={focusTrapped.value}>
+        <label className="label">
+          <Checkbox state={focusTrapped} className="checkbox" />
+          Trap focus
+        </label>
+        <Button className="button">Button 1</Button>
+        <Button className="button">Button 2</Button>
+        <Button className="button" disabled>
+          Button 3
+        </Button>
+        <input className="input" title="one" placeholder="Input" />
+        <input
+          className="input"
+          title="two"
+          disabled
+          placeholder="Disabled input"
+        />
+      </FocusTrapRegion>
+    </>
+  );
+}
```

Comment:
Can we make it so the first or the last tabbable elements are lazily mounted (for example, only when `enabled === true`)? The use case is that dialogs may render their contents only when the dialog is open.

Reply:
I just want to make sure we're covering the case where tabbable elements are lazily mounted within the focus trap region, and they could be the first/last tabbable elements.

Reply:
Okay! I got it now. Will look into it.